docs/user_guide/transformation/PowerTransformer.rst

```diff
@@ -5,66 +5,349 @@
 PowerTransformer
 ================
 
-The :class:`PowerTransformer()` applies power or exponential transformations to numerical
-variables.
+Power transformations are a family of mathematical functions used to transform numerical variables into a more suitable shape for modeling. The transformation function is typically represented as :math:`x' = x^\\lambda`, where :math:`x` is the original variable and :math:`\\lambda` is the transformation parameter.
 
-Let's load the house prices dataset and  separate it into train and test sets (more
-details about the dataset :ref:`here <datasets>`).
+These transformations help stabilize the variance, make the data adopt a more normal distribution-like shape, and/or improve the linearity of relationships.
+
+Use of Power transformations
+----------------------------
+
+Power transformations are particularly useful for meeting the assumptions of statistical tests, and models that require linear relationships between variables and homoscedasticity (constant variance across values). They can also help in reducing skewness in the data, i.e., by normalizing distributions.
+
+Power transformations differ from scalers in that they modify the distribution of the data, typically to stabilize variance and normalize the distribution, whereas scalers simply adjust the scale of the data without altering its underlying distribution.
+
+In short, power functions provide an excellent data analysis toolkit, especially for linear models (regression or classification).
+
+Special cases of power transformations
+--------------------------------------
+
+Most variable transformations, like the logarithm, the reciprocal and the square root, are special cases of power transformations, where the exponent (lambda) is 0, -1 and 0.5, respectively.
+
+You can apply these transformations with :class:`PowerTransformer()`, as we will see later in this page, or through dedicated transformers, like :class:`LogarithmTransformer` and :class:`ReciprocalTransformer`.
```

Comment:
It's Logtransformer instead of LogarithmTransformer, that's why it does not link to the corresponding page

Reply:
Fixed, I'll update the PR soon!